sci-libs/shapelib/files/0012-dbfadd.c-dbfcreate.c-run-pre-commit.patch

```diff
@@ -0,0 +1,64 @@
+From 5316df98308c76a41a1a3d5cbc6731caea249ec3 Mon Sep 17 00:00:00 2001
```

Comment:
This patch definitely isn't normally right for backporting _but_ I could allow it if it makes backporting a later patch (that we actually need) easier.